lib/python/synchrotronRadiationExtension/synchrotronRequirements.py

```diff
@@ -0,0 +1,98 @@
+# /* Copyright 2024 Filip Optolowicz
+#  *
+#  * This file is part of PIConGPU.
+#  *
+#  * PIConGPU is free software: you can redistribute it and/or modify
+#  * it under the terms of the GNU General Public License as published by
+#  * the Free Software Foundation, either version 3 of the License, or
+#  * (at your option) any later version.
+#  *
+#  * PIConGPU is distributed in the hope that it will be useful,
+#  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  * GNU General Public License for more details.
+#  *
+#  * You should have received a copy of the GNU General Public License
+#  * along with PIConGPU.
+#  * If not, see <http://www.gnu.org/licenses/>.
+#  */
+
+import numpy as np
+import scipy.constants as const
+
+# Constants
+e = const.elementary_charge  # Elementary charge in C
+m_e = const.m_e  # Electron mass in kg
+c = const.c  # Speed of light in m/s
+hbar = const.hbar  # Reduced Planck constant in J/s
+miu0 = const.mu_0  # Vacuum permeability in H/m
+eps0 = const.epsilon_0  # Vacuum permittivity in F/m
```

Comment:
```suggestion
from scipy.constants import elementary_charge as e, m_e, c, hbar, mu_0 as miu0, epsilon_0 as eps0

```
of course similarly for the other files.